### PR TITLE
Point at a branch of blacklight that supports jquery3. Ref #236

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -67,6 +67,7 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
+gem 'blacklight', github: 'projectblacklight/blacklight', branch: 'release-6.x'
```
Comment (cbeer) [inline Gemfile]: Can we just cut a new release?

Comment (jcoyne) [reply Gemfile]: Sure, are we done yet though?

Comment (cbeer) [reply Gemfile]: Done enough, no?

Comment (jcoyne) [reply Gemfile]: What about #222?

**Pull request closed without merging**